### ISSUE TITLE
Missing icons

### ISSUE THREAD

**Issue by BinTofajjal** [JanDeDobbeleer/oh-my-posh#694]
[Screenshot_35](https://user-images.githubusercontent.com/66693938/116678704-72897b80-a9cb-11eb-8a4c-48bf04025c78.png)
Icons are missing...

**Comment by BinTofajjal** [comment]
Solved - https://www.nerdfonts.com/font-downloads

**Comment by attisan** [comment]
I'm using fira code, still have missing symbols (using putty)

**Comment by JanDeDobbeleer** [comment]
@attisan did you set the font correctly in Putty? We have a [page](https://ohmyposh.dev/docs/fonts) on how to install the fonts correctly on Windows.

**Comment by attisan** [comment]
though I did (some parts did work, others didn't show as expected) I ended up switching to MesloLGM 🤷‍♂️ (thx @JanDeDobbeleer)

**Comment by JanDeDobbeleer** [comment]
@attisan you can also always create your own version of the theme. That way you can use any font you want.

**Comment by nistre** [comment]
@JanDeDobbeleer do you have an updated link?  That page you linked above doesn't work anymore but it might resolve an issue I'm having?

**Comment by JanDeDobbeleer** [comment]
@nistre right [here](https://ohmyposh.dev/docs/installation/fonts).

**Comment by nistre** [comment]
Gotcha, I just found that myself, haha.  Though I'm having an issue getting the vscode terminal to use the CascadiaCode font I just installed, or if I can make both the font and posh them global that would work too

**Comment by JanDeDobbeleer** [comment]
You need to set the font in every terminal yourself, that's a terminal setting. The theme can be set using the --config flag and added to the shell's configuration mm(rc or profile scripts).

**Comment by nistre** [comment]
Gotcha.  I had followed previously followed the [PowerShell Migration](https://ohmyposh.dev/docs/migrating) in setting the theme.  And just followed a couple other this morning.  After installing a recommended Meslo LGM NF download from the Posh Fonts page (any font I installed via the `oh-my-posh font install` command wasn't recognized despite seeing the font in other applications such as Word).  After installing the direct download, then I was able to set and use the font and font-size via the settings.json values:
    "terminal.integrated.fontFamily": "MesloLGMDZ NF",
    "terminal.integrated.fontSize": 12,
    
Thanks @JanDeDobbeleer for your help and work!
